scriptmodules/libretrocores/lr-mesen.sh

```diff
@@ -13,11 +13,12 @@ rp_module_id="lr-mesen"
 rp_module_desc="High-accuracy NES and Famicom emulator"
 rp_module_help="ROM Extensions: .nes .fds .unf .unif .zip\n\nCopy your NES roms to $romdir/nes\nFamicom roms to $romdir/fds\nCopy the recommended BIOS file disksys.rom to $biosdir"
 rp_module_licence="GPL3 https://raw.githubusercontent.com/sourmesen/mesen/master/LICENSE"
+rp_module_repo="git https://github.com/SourMesen/Mesen.git master"
```

Comment:
(Unrelated to this PR) Maybe switch to https://github.com/libretro/Mesen.
Original repository has been archived (see the author's [announcement](https://old.reddit.com/r/emulation/comments/j58hfu/mesen_and_mesens_github_repositories_are_now/g7sti7q/)).

Reply:
Have merged this into the original commit but have added it as an addition commit msg note.